settings.json

```diff
@@ -601,6 +601,14 @@
       ]
     }
   ],
+  "prisma": [
+    {
+      "command": "prisma-language-server",
+      "requires": [
+        "npm"
+      ]
+    }
+  ],
```

Comment:
I think we may add vim-prisma to settings.json
```json
      "vim_plugin": {
        "extensions": [
          "prisma"
        ],
        "name": "pantharshit00/vim-prisma"
      }
```

Reply:
Thank you! I added them.